libraries/stdlib/samples/test/samples/text/strings.kt

```diff
@@ -17,6 +17,15 @@ class Strings {
         assertPrints("Abcd".decapitalize(), "abcd")
     }
 
+    @Sample
+    fun replaceFirstChar() {
+        assertPrints("kotlin".replaceFirstChar { it.uppercaseChar() }, "Kotlin")
```

Comment:
`it.uppercase()` would be a more common usage example.

Reply:
done